### ISSUE TITLE
Github Action to autodeploy to wpengine

### ISSUE THREAD

**Issue by ccorda** [patronage/bubs-next#32]
When a commit comes to master, and there are files changed in the `wordpress` folder (but not only in `website`), can we trigger a GH action that runs our build script to deploy?

**Comment by kylehotchkiss** [comment]
Is this something we're sure we want? I often prefer to manually deploy from computer, sometimes even deploying a PR to WP prod shortly before merging so the vercel build won't break due to graphql changes

**Comment by ccorda** [comment]
I agree it's complicated. I'd like to at least see docs about how to do it manually, then we could revisit whether it's automatable or not.

**Comment by ccorda** [comment]
Updated title to reflect this is about documentation, not a Github Action:

[Internal link for WIP docs](https://www.notion.so/patronage/Deployment-8c9e2a09e773458f8bdd1cc3f2a8d657).
